doc/developer/entrypoints.rst

```diff
@@ -81,7 +76,12 @@ or (if ``myapp`` is not in your ``PYTHONPATH``)::
 
     python path/to/myapp do-my-thing --post-process settingsFile.yaml
 
-.. tip:: The settings file will be read into a ``Settings`` object. This ``Settings``
-    object will be passed widely around the code. Please do not edit these settings
-    during a run. The idea of "run settings" is a lot simpler to understand when they
-    don't change. And such changes tend to hide data from other developers.
+.. tip:: The settings file will be read into a ``Settings`` object. This ``Settings`` object will be passed widely
+    around the code. Please do not edit these settings during a run. The idea of "run settings" is a lot simpler to
+    understand when they don't change. And such changes tend to hide data from other developers.
+
+Every ``ArmiPlugin`` can subclass the ``defineEntryPoints`` method. This is the place to add all your entry points. ARMI
+comes with an extensive :py:class:`~armi.cli.EntryPointsPlugin` that comes with several CLI entry points that you will
+want. It is important to note that if you are building your own ARMI ``Application``, if you do not include the
+``EntryPointsPlugin``, you will not have these entry points. But that is fine if you do not want them or if only want
+some of them you can build your own list in a custom ``defineEntryPoints`` method.
```

Comment:
The last sentence needs a little grammar love here. But also I have questions!

I have experience with an app that only defines `defineEntryPoints` and those entrypoints subclass ARMI's `EntryPoint`. So it doesn't have the `EntryPoint` plugin registered. That app does have access to ARMI's `run` entrypoint via `<app> run`. So i'm not sure what you're saying here is true (unsure if that's a bug, I'm misunderstanding what you're saying, or something else)

Reply:
Ah Ok since we do `from armi.apps import App; MyApp(App)` we get this registered. Cool cool. Ok I'll resolve this then go suggest some rewording in this paragraph